store/src/java/com/zimbra/qa/unittest/TestConversion.java

```diff
@@ -19,43 +19,61 @@
 
 import java.io.File;
 
-import junit.framework.TestCase;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
 
-import com.zimbra.common.localconfig.LC;
-import com.zimbra.common.util.ByteUtil;
 import com.zimbra.client.ZMailbox;
 import com.zimbra.client.ZMessage;
 import com.zimbra.client.ZMessage.ZMimePart;
+import com.zimbra.common.localconfig.LC;
+import com.zimbra.common.util.ByteUtil;
 
-public class TestConversion extends TestCase {
+public class TestConversion {
 
-    private static final String USER_NAME = "user1";
+    @Rule
+    public TestName testInfo = new TestName();
+    private String testName = null;
+    private String USER_NAME = null;
     private static final String NAME_PREFIX = TestConversion.class.getSimpleName();
 
-    @Override
-    public void setUp()
-    throws Exception {
-        cleanUp();
+    @Before
+    public void setUp() throws Exception {
+        testName = testInfo.getMethodName();
+        USER_NAME = NAME_PREFIX + "-" + testName + "-user";
+        TestUtil.createAccount(USER_NAME);
```

Comment:
As with the other two requests, why don't we make all tests cleanup before the start in case a previous run was interrupted by a crash.